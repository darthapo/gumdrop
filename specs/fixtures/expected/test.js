(function() {
  var Test;

  Test = (function() {

    function Test() {}

    Test.VERSION = "1.0";

    return Test;

  })();

}).call(this);
