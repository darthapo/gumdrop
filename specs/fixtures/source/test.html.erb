<% set :layout, nil %><% self.crappy= true %><h1>Test</h1>
<p><%= "Hello" %> <%= render 'user_sets_layout' %>.</p><% set name:"bob", age:"stuff" %>