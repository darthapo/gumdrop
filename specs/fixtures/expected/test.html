<h1>Test</h1>
<p>Hello <div>User</div>.</p>